package-lock.json

```diff
@@ -1,12 +1,11 @@
 {
   "name": "identity-style-guide",
-  "version": "6.4.1",
+  "version": "6.4.2",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
-      "name": "identity-style-guide",
```

Comment:
Can you check that your version of `npm` is up to date? I think it would not be expected that this line is removed, and I think I've seen it only in older versions of npm.

Reply:
I was on npm v7.18.1, and node v16.4.2. 
Should I update and publish again or do I need to delete the branch and start again?

Reply:
No need to publish again. I think if you update npm, then run `npm install` on this branch, you should see a change that you just need to commit.

Reply:
We also have a lint for this, which is currently failing.

>Error: There are uncommitted changes after running 'npm install'
>make: *** [Makefile:28: validate-package-lock] Error 1

https://app.circleci.com/pipelines/github/18F/identity-style-guide/1025/workflows/b4fcfac5-1f14-47e0-8a69-4fb71cbd51bc/jobs/2760

Reply:
fixed